worklog.go

```diff
@@ -0,0 +1,111 @@
+package jira
+
+import (
+	"time"
+	"strings"
+	"net/url"
+	"fmt"
+)
+
+const TTWorklogTimeFormat = "2006-01-02T15:04:05.000"
+const TTWorklogDateFormat = "2006-01-02"
```

Comment:
Can you add comments here? With a link to the docs as well?

Reply:
WorklogBean that returns worklog api uses this time format. Watch the `Returns` section: http://developer.tempo.io/doc/timesheets/api/rest/latest/#848933329
![2018-05-20 10 46 24](https://user-images.githubusercontent.com/128070/40276760-1ced76b4-5c1b-11e8-9b96-209681d17f83.png)